package.json

```diff
@@ -12,6 +12,7 @@
     "type": "git",
     "url": "git+https://github.com/HonkingGoose/vale-plain-language-guidelines.git"
   },
+  "license": "AGPL-3.0-only",
```

Comment:
I think the license file itself allows people to use `3.0.` or later, so maybe we need to change things.